shells/android/java/arcs/crdt/parcelables/ParcelableCrdtSet.kt

```diff
@@ -75,7 +75,7 @@ object ParcelableCrdtSet {
                         }.actual
```

Comment:
Looks like another opportunity to move the `.actual` within `the requireNotNull`. Sorry I missed this on the first go-round.

Reply:
Done